src/services/sync/strategies/TSP/TSPSchoolSyncer.js

```diff
@@ -13,6 +14,7 @@ const { switchSchool, getInvalidatedUuid } = require('./SchoolChange');
 const SYNCER_TARGET = 'tsp-school';
 const schoolLimit = pLimit(Configuration.get('TSP_SCHOOL_SYNCER__SCHOOL_LIMIT'));
 const limit = pLimit(Configuration.get('TSP_SCHOOL_SYNCER__STUDENTS_TEACHERS_CLASSES_LIMIT'));
+const lastSyncedAtEnabled = Configuration.get('FEATURE_SYNC_LAST_SYNCED_AT_ENABLED') === true;
```

Comment:
I think was some kind of argument to not call the config like this, but rather in the class constructor or somewhere when it is used

Reply:
I did it like that as it's common across the legacy server modules so I wanted to stick to make it consistent. But I agree with you, changed that in [this commit](https://github.com/hpi-schul-cloud/schulcloud-server/pull/4221/commits/f6a25f6a2344b1d19a351ae7fa2a8903389be066).